governance/pm-core-meeting-notes/24Jan24.md

```diff
@@ -0,0 +1,238 @@
+# The Turing Way - Core Project Management Meetings 24 January
+
+## Bi-weekly meetings (with some exceptions)
+
+**Participants** 
+This call is joined by *The Turing Way* Core Staff:
+
+* Kirstie Whitaker
+* Malvika Sharan 
+* Alexandra Araujo Alvarez 
+* Arielle Bennett 
+
+## Agenda
+
+Review together the agenda and agree on priorities and timings.  
+
+:hourglass_flowing_sand: **Timing:** Total 90 min
+
+| Duration | Activity |
+| ---- | -------- |
+| Start | 👋 Welcome |
+| 05 mins | Check in   |
+| 20 mins | Infrastructure WG   
+| 10 mins | TTW Community Call Agenda (15 Feb)
+| 20 min | Collaborations Workshop (TPS & TTW Participation) 
+| 10 min | CZI Localisation Proposal / Jupyterhub updates
+| 20 mins | Other
+5 mins | Wrap up  
+
```

Comment:
## Actions from the meeting:
- [ ] (AAA): update the Domain request with all the proposal - https://github.com/the-turing-way/the-turing-way/issues/3266
- [ ] (MS) New Working Group - Communications WG —> MS suggested to open an Expression of interest to understand and explain what is needed to create this wg, who has capacity to do it? MS will think about it and will bring an ask for a proposal of Expression of interest to the next meeting
- [X] (AAA) Ask the infrastructure wg to open an issue for guidance to maintain a light-weight repo - (https://github.com/the-turing-way/the-turing-way/issues/3516)
- [ ] (ALS/AAA) Start an issue with "Best design of a working group repo, how we want them to be structured, what is needed, easy templates of the minimum they require to facilitate contributions
- [ ] (ALS/AAA) best design to align the community to navigate on the different repos (issue)